package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Nov  7 10:08:49 UTC 2018 - dgonzalez@suse.com
+
+- Allow to proposal work on top of existing MD RAIDs
+  (fate#326573).
```

Comment:
I am not sure if the sentence is correct. I guess it should be: "Allow to the proposal to work on top ..."

My proposal: "Improve the proposal to be able to work with existing MD RAIDs ((fate#326573).)"